mne_bids/tests/test_read.py

```diff
@@ -787,12 +787,55 @@ def test_write_read_fif_split_file():
 
     bids_path.update(split='01')
     raw2 = read_raw_bids(bids_path=bids_path)
+
     bids_path.update(split='02')
     raw3 = read_raw_bids(bids_path=bids_path)
     assert len(raw) == len(raw1)
     assert len(raw) == len(raw2)
     assert len(raw) > len(raw3)
 
+    # check that the scans list contains two scans
+    scans_tsv = BIDSPath(
+        subject=subject_id, session=session_id,
+        suffix='scans', extension='.tsv',
+        root=bids_root)
+    scan_data = _from_tsv(scans_tsv)
+
+    scan_fnames = scan_data['filename']
+    scan_acqtime = scan_data['acq_time']
+    assert len(scan_fnames) == 2
+    assert 'split-01' in scan_fnames[0] and 'split-02' in scan_fnames[1]
+    # check that the acq_times in scans.tsv are the same
+    assert scan_acqtime[0] == scan_acqtime[1]
+    # check the recordings are in the correct order
+    assert raw2.first_time < raw3.first_time
+
```

Comment:
Now, the scans.tsv have the correct `acq_time` metadata, which is tested. You could then say manually alter the scans.tsv file to not have the same acq_time for one of the split files:

E.g.
```Python
buff_scan_data = scan_data.copy()  # create a copy, so that way you can reset afterwards
scan_data['acq_time'][<one of the split indices>] = <some other datetime>
_to_tsv(scan_data, scans_tsv)
with pytest.raises(ValueError, match='split files must have the same acq_time'):
    read_raw_bids(bids_path)
# reset scans.tsv file for downstream tests
_to_tsv(buff_scan_data, scans_tsv)
```

Reply:
Thanks, that helped a lot!